mzLib/MzLibUtil/MrsNoiseEstimation/MRSNoiseEstimator.cs

```diff
@@ -48,7 +46,7 @@ public static class MRSNoiseEstimator
                 // and the standard deviation of the noise at each level. 
                 // 5. Select all points that belong to the noise; they don't have an significant coefficients above noise 
                 var booleanizedLevels = BooleanizeLevels(wtOutput,
-                    stdevPrevious, 1.97); 
+                    estimatedStandardDeviation, 1.97); 
```

Comment:
Why 1.97? Seems like a magic number at first glance

Reply:
Added comment for magic number in latest commit.